util/config/run-in-venv-with-python-bindings.bash

```diff
@@ -17,4 +17,8 @@ if [ ! -d "$chpl_frontend_py_deps/chapel" ]; then
   exit 1
 fi
 
+if [ "$CHPL_SANITIZE" == "address" ]; then
+  export LD_PRELOAD=$(gcc -print-file-name=libasan.so)
```

Comment:
I would also recommend you prevent users from calling this on MacOS, similar to the `setup.py` changes. The logic should be duplicated because while `export CHPL_SANITIZE=address` is enough to trigger a rebuild for the compiler/runtime, its not for `chapel-py`. So a user could accidentally have an asan compiler on MacOS and then this logic will produce unexpected results